src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -2,18 +2,48 @@
 
 <Project>
  
-  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
-  
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+  <UsingTask TaskName="GetLastStablePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+
   <Target Name="RunPackageValidation" AfterTargets="Pack">
+    
+    <GetLastStablePackage PackageId="$(PackageId)"
+      LocalPackagesPath="$(NuGetPackageRoot)$(PreviousPackageName)"
```

Comment:
What happens in `$(PreviousPackageName)` isn't set?

Reply:
Do we need to set this ourselves in dotnet/runtime as well? Or is there a default somewhere?

Reply:
Also, shouldn't this logic under a switch or do we always want to validate against the previous package?